apps/src/util/experiments.js

```diff
@@ -28,6 +28,8 @@ experiments.TEACHER_DASHBOARD_SECTION_BUTTONS =
 experiments.TEACHER_DASHBOARD_SECTION_BUTTONS_ALTERNATE_TEXT =
   'teacher-dashboard-section-buttons-alternate-text';
 experiments.MINI_TOOLBOX = 'miniToolbox';
+experiments.VIRTUAL_WORKSHOPS = 'virtual-workshops';
```

Comment:
is this line purposeful? it looks unrelated

Reply:
it was a merge conflict I had to resolve

Reply:
this was a merge conflict I had to resolve